Shared/Samples/Authenticate with Integrated Windows Authentication/README.md

```diff
@@ -0,0 +1,48 @@
+# Authenticate with Integrated Windows Authentication
+
+Connect to an IWA secured Portal and search for maps.
+
+![Image of authenticate with integrated windows authentication](authenticate-with-integrated-windows-authentication-1.png)
+
+## Use case
+
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
+
+## How to use the sample
+
+1. Enter the URL to your IWA-secured portal.
+2. Click the connect button to search for web maps stored on the portal.
+3. You will be prompted for a user name, and password.
+4. If you authenticate successfully, portal item results will display in the list.
+5. Select a web map item to display it in the map view.
+
+## How it works
+
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
```

Comment:
It does not give separate option to provide the `domain`. If they have to provide the domain then it should be part of the username`domain\\username`
```suggestion
1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username and password) if a secure resource is encountered.
```